### PR TITLE
Add note about YouTube links not handled in group chats to README

### PR DESCRIPTION
**Pull request by AleksandrSl**
I spent 10 minutes wondering why YouTube doesn't have any effect 😃. I think it's worth mentioning in the docs. 

Submitted with [Web Publisher](https://developer.stackblitz.com/codeflow/integrating-web-publisher)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ Currently the following services are supported:
   - Apple Music
   - Tidal
 
+Note that bot doesn't react to YouTube links in group chats. Reasons are [here](9dogs/tg-odesli-bot/blob/master/README.md).
```
Comment (AlexWayfer) [inline README.md:56]: The link is broken. And where it should lead?

**Review by AlexWayfer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (9dogs) [reply README.md:56]: I think we can link this issue: https://github.com/9dogs/tg-odesli-bot/issues/12

Comment (AleksandrSl) [reply README.md:56]: Yeah sorry, didn't spot that I pasted the wrong link. It was supposed to lead to the issue mentioned

**Review by AlexWayfer — APPROVED** [review]
Now it's OK.

**Review by 9dogs — APPROVED** [review]

**Pull request merged**